articles/iot-edge/how-to-publish-subscribe.md

```diff
@@ -28,7 +28,7 @@ You can use Azure IoT Edge MQTT broker to publish and subscribe messages. This a
 - Have an **IoT Edge device with version 1.2 or above**. Since IoT Edge MQTT broker is currently in public preview, set the following environment variables to true on the edgeHub container to enable the MQTT broker:
 
     - experimentalFeatures__enabled
-    - mqttbroker__enabled
+    - experimentalFeatures__mqttBrokerEnabled
```

Comment:
This looks good, but can we instead put it in this format? 
https://github.com/MicrosoftDocs/azure-docs/blob/master/articles/iot-edge/how-to-connect-downstream-iot-edge-device.md#configure-runtime-modules-for-public-preview